spec/features/admin/index_spec.rb

```diff
@@ -66,13 +76,22 @@
 
     it "should displays a section for incomplete invoices with ids as links" do
       visit admin_index_path
-      
+
       within "#incomplete-invoices" do
         expect(page).to have_content("Incomplete Invoices")
         expect(page).to have_content("Invoice ##{invoice_7.id}")
         expect(page).to have_link("#{invoice_7.id}", :href => admin_invoice_path(invoice_7))
         expect(page).to_not have_link("#{invoice_8.id}", :href => admin_invoice_path(invoice_8))
       end
     end
+
+    it "should list invoice dates and sort them by oldest to newest" do
+      visit admin_index_path
+
+      within "#incomplete-invoices" do
+        expect(invoice_10.format_time_stamp).to appear_before(invoice_9.format_time_stamp)
```

Comment:
orderly is tricky and it's best to make the test that compares the same called attributes in the expect statement. ex: expect(okay_1.yes). to appear_before(okay_2.yes)